src/Uno.UI/DataBinding/BindingPropertyHelper.FastConvert.cs

```diff
@@ -112,6 +97,17 @@ private static bool FastColorOffsetConvert(Type outputType, ColorOffset input, r
 			return false;
 		}
 
+		private static bool FastSolidColorBrushConvert(Type outputType, SolidColorBrush solidColorBrush,
```

Comment:
Is this something WinUI does? I find it very surprising. In what circumstance is it legal to supply a `SolidColorBrush` to a binding when WinUI is expecting a `Color`?

Reply:
@davidjohnoliver Check this: https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/ProgressBar/ProgressBar.xaml#L47

It's binding a `SolidColorBrush` to a `Color?` property.